onedal/basic_statistics/tests/test_basic_statistics.py

```diff
@@ -32,30 +32,12 @@
         ("standard_deviation", np.std, (3e-5, 3e-5)),
     ]
 
-    @pytest.mark.parametrize("queue", get_queues())
-    @pytest.mark.parametrize("dtype", [np.float32, np.float64])
-    def test_basic_uniform(queue, dtype):
-        seed = 42
-        s_count, f_count = 70000, 29
-
-        gen = np.random.default_rng(seed)
-        data = gen.uniform(low=-0.5, high=+0.6, size=(s_count, f_count))
-        data = data.astype(dtype=dtype)
-
-        alg = BasicStatistics(result_options="mean")
-        res = alg.compute(data, queue=queue)
-
-        res_mean = res["mean"]
-        gtr_mean = np.mean(data, axis=0)
-        tol = 2e-5 if res_mean.dtype == np.float32 else 1e-7
-        assert_allclose(gtr_mean, res_mean, rtol=tol)
-
     @pytest.mark.parametrize("queue", get_queues())
     @pytest.mark.parametrize("option", options_and_tests)
     @pytest.mark.parametrize("dtype", [np.float32, np.float64])
     def test_option_uniform(queue, option, dtype):
         seed = 77
-        s_count, f_count = 19999, 31
+        s_count, f_count = 1000, 31
```

Comment:
I believe we can afford testing such a simple functionality on a larger datasets

Reply:
your comment is again in contradiction with @samir-nasibli 's one. as for me it does not really matter to be honest

Reply:
but have to say that the test starts failing after the change.

Reply:
> your comment is again in contradiction with @samir-nasibli 's one. as for me it does not really matter to be honest

Sorry, where is the contradiction? this number is really big for a test suit.

Reply:
please read carefully

Reply:
@olegkkruglov if your are refering this comment: https://github.com/intel/scikit-learn-intelex/pull/1644#discussion_r1448139817
It is a question, not a proposal change :)